scripts/scil_compute_NODDI_priors.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Compute the axial and mean diffusivity priors for NODDI.
+"""
+
+from __future__ import division, print_function
+import argparse
+import logging
+
+import nibabel as nib
+import numpy as np
+
+from scilpy.io.utils import (assert_inputs_exist,
+                             assert_outputs_exist,
+                             add_overwrite_arg,
+                             add_verbose_arg)
+
+EPILOG = """
+Reference:
+    [1] Zhang H, Schneider T, Wheeler-Kingshott CA, Alexander DC.
+        NODDI: practical in vivo neurite orientation dispersion
+        and density imaging of the human brain.
+        NeuroImage. 2012 Jul 16;61:1000-16.
+"""
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
```

Comment:
This parser could use some groups to collect together arguments in related categories